ert3/evaluator/_evaluator.py

```diff
@@ -50,8 +50,9 @@ def _run(
 
 def evaluate(
     ensemble: Ensemble,
+    custom_port_range: Optional[range] = None,
```

Comment:
Are we to expose `port_range` as parameter here?

Reply:
I dunno. Any reason why not?

Reply:
You might be right though as I can't see any other way to deliver the port range there.

Reply:
I agree with both of you.

Reply:
:monkey: :monkey: :monkey: :monkey: :question: